Python/Product/Analysis/Infrastructure/UriEqualityComparer.cs

```diff
@@ -18,7 +18,7 @@
 using System.Collections.Generic;
 
 namespace Microsoft.PythonTools.Analysis.Infrastructure {
-    sealed class UriEqualityComparer : IEqualityComparer<Uri> {
+    public sealed class UriEqualityComparer : IEqualityComparer<Uri> {
```

Comment:
Small types like `UriEqualityComparer` can be copied to PTVS-assemly instead of being public.

Reply:
This one is only used in PythonTools project, we could move it there.

Reply:
Done